src/Compatibility/Microsoft.DotNet.PackageValidation/IPackageLogger.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.Build.Framework;
-using Microsoft.DotNet.ApiCompatibility.Abstractions;
 using Microsoft.DotNet.Compatibility.ErrorSuppression;
 
 namespace Microsoft.DotNet.PackageValidation
```

Comment:
Move this file as well and rename it as the API Compat task will probably end up using the logger?

Reply:
The would be a circular dependency Compatibility depends on package validation & api compatibility. we can move the type down to api compat.

Reply:
Hmm I don't think API Compat will use this interface, just the logger class. Let's leave it where it is and we can move it down if needed later on.